src/client/keycode.cpp

```diff
@@ -45,7 +45,7 @@ struct table_key {
 
 #define N_(text) text
 
-static const struct table_key table[] = {
+static std::vector<table_key> table = {
```

Comment:
fwiw bringing this code to modern standards would be much appreciated, even if it doesn't solve the issue at hand